test/commands/platform-dependency-version.test.ts

```diff
@@ -95,12 +95,30 @@ describe('platform-dependency-version', () => {
 
   // Test WC versions
   describe('with `--wcVersion` param', () => {
-    describe('checks dependencies versions for a given WP & WC versions', () => {
+    describe('checks dependencies versions for a given WC version', () => {
+      test
+      .stdout()
+      .command(['platform-dependency-version', '--wcVersion=7.0.1', '--dependenciesJSON=test/mocks/.externalized.json'])
+      .it('renders it in a table', ctx => {
+        // Header contains the WC version.
+        expect(ctx.stdout).to.have.string('WooCommerce 7.0.1')
+
+        // Should not include WP versions
+        expect(ctx.stdout).to.match(/@wordpress\/data\s*$/ms)
+        expect(ctx.stdout).to.match(/react\s*$/ms)
+        // Prints packages with their versions.
+        expect(ctx.stdout).to.match(/foo\s*$/ms) // No version for unmatched package
```

Comment:
There is no other package with a version to be asserted, so the two lines of code comments look a bit conflicting. Maybe they could be adjusted.